pkg/cloudprovider/vsphereparavirtual/cloud.go

```diff
@@ -111,6 +111,16 @@ func (cp *VSphereParavirtual) Initialize(clientBuilder cloudprovider.ControllerC
 		klog.Fatalf("Failed to get cluster namespace: %v", err)
 	}
 
+	nodeLister := cp.informMgr.GetNodeLister()
+	routes, err := NewRoutes(clusterNS, kcfg, *cp.ownerReference, cp.client, nodeLister)
```

Comment:
Do we have a flag in cloud provider to enable/disable this feature?

Reply:
yes, if `controllers=route` is set, then we would enable route controller and its corresponding logic